<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentDialog</class>
 <widget class="QDialog" name="DocumentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>369</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>369</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add / Edit Document</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename_edit">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>40</y>
     <width>266</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_edit">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>165</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="note_edit">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>255</y>
     <width>266</width>
     <height>106</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>25</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>105</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Creation Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_file_button">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>38</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select FIle To Upload</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/select.png</normaloff>:/wnt/select.png</iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="author_cbox">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>120</y>
     <width>266</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>65</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Document Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doc_no_edit">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>80</y>
     <width>266</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="doc_type_cbox">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>210</y>
     <width>266</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>195</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Document Type</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>filename_edit</tabstop>
  <tabstop>select_file_button</tabstop>
  <tabstop>doc_no_edit</tabstop>
  <tabstop>author_cbox</tabstop>
  <tabstop>date_edit</tabstop>
  <tabstop>doc_type_cbox</tabstop>
  <tabstop>note_edit</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DocumentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DocumentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
